ext/gd/tests/gdimage_cloning.phpt

```diff
@@ -0,0 +1,26 @@
+--TEST--
+Checks that GdImage instances can be cloned from gd > 2.2.0
+--SKIPIF--
+<?php if (version_compare(GD_VERSION, '2.2.0', '<')) { die("Skipped: GdImage is not clonable before gd 2.2.0"); } ?>
+--EXTENSIONS--
+gd
+--FILE--
+<?php
+
+    $foo = imageCreateTrueColor(32, 32);
```

Comment:
Nit: Shouldn't be indented

Reply:
Indeed. I also suggest to use the canonical function names for test (i.e. in this case lowercase).